addons/website/static/tests/tours/snippet_rating.js

```diff
@@ -0,0 +1,33 @@
+/** @odoo-module **/
+import wTourUtils from '@website/js/tours/tour_utils';
+
+wTourUtils.registerWebsitePreviewTour('snippet_rating', {
+    test: true,
+    url: '/',
+    edition: true,
+}, () => [
+    wTourUtils.dragNDrop({ id: 's_rating', name: 'Rating' }),
+    wTourUtils.clickOnSnippet({ id: 's_rating', name: 'Rating' }),
+    wTourUtils.changeOption('Rating', 'we-select:has([data-select-class]) we-toggler'),
+    wTourUtils.changeOption('Rating', 'we-button[data-select-class="s_rating_inline"]'),
+    {
+        content: "Check whether s_rating_inline class applied or not",
+        trigger: ":iframe .s_rating_inline",
+        isCheck: true
+    },
+    wTourUtils.changeOption('Rating', 'we-select:has([data-select-class]) we-toggler'),
+    wTourUtils.changeOption('Rating', 'we-button[data-select-class="s_rating_none"]'),
+    {
+        content: "Check whether s_rating_none class applied or not",
+        trigger: ":iframe .s_rating_none",
+        isCheck: true,
+    },
+    wTourUtils.changeOption('Rating', 'we-select:has([data-select-class]) we-toggler'),
+    wTourUtils.changeOption('Rating', 'we-button[data-select-class=""]'),
+    {
+        content: "Check whether s_rating_none class removed or not",
+        trigger: ":not(iframe .s_rating_none)",
```

Comment:
This is always `true`, please test that the tour fails without the previous step.

Reply:
I tested the tour without the steps:
```js
    wTourUtils.changeOption('Rating', 'we-select:has([data-select-class]) we-toggler'),
    wTourUtils.changeOption('Rating', 'we-button[data-select-class=""]'),
 ```
 And it didn' t fail as It should ..

Reply:
@bvr-odoo I got the point, `trigger: ":not(iframe .s_rating_none)",` this line was not working as i thought it would. 
I found another way to check whether class `s_rating_none` (or `s_rating_no_title`, as per new naming) is removed or not using custom run function.